apps/backend/src/db/migrations/20250807194934_add_avatar_columns_to_user_details.ts

```diff
@@ -0,0 +1,26 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	USER_DETAILS: "user_details",
+} as const;
+
+const ColumnName = {
+	AVATAR_KEY: "avatar_key",
+	AVATAR_URL: "avatar_url",
+} as const;
+
+async function down(knex: Knex): Promise<void> {
+	await knex.schema.alterTable(TableName.USER_DETAILS, (table) => {
+		table.dropColumn(ColumnName.AVATAR_KEY);
+		table.dropColumn(ColumnName.AVATAR_URL);
+	});
+}
+
+async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable(TableName.USER_DETAILS, (table) => {
+		table.string(ColumnName.AVATAR_KEY);
+		table.string(ColumnName.AVATAR_URL);
+	});
+}
```

Comment:
We don't need avatarKey and avatarUrl in userDetails. We should have url and key in files table